.golangci.yml

```diff
@@ -58,6 +58,11 @@ linters-settings:
       - int
       - make
       - '\.WithTimeout'
+  revive:
+    rules:
+      - name: exported
+        arguments:
+          - disableStutteringCheck
```

Comment:
keeping `disableStutteringCheck` to focus on functions only in this PR. In future PRs, we can consider removing this line and enable extra checks (for types, vars, etc)